src/config.ts

```diff
@@ -0,0 +1,8 @@
+export const introspectionStorageAccessKey: string = process.env
+  .INTROSPECTION_STORAGE_ACCESS_KEY!;
+export const introspectionStorageAccountName: string = process.env
```

Comment:
Non-sensitive values can be configured directly in config.yaml. Sensitive values such as storage account key, service principal id should be configured in .env file.

Reply:
Yup agreed, I've removed this file already